code/API_definitions/kyc-match.yaml

```
@@ -0,0 +1,536 @@
+openapi: 3.0.3
+info:
+  title: Know Your Customer Match
+
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+
+  version: 0.1.0-wip
+
+servers:
+  - url: '{apiRoot}/{basePath}'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: kyc-match/v0
+        description: Base path for the KYC-Match API
+
+tags:
+  - name: KYC_Match

```

I think that adding KYC is a bit redundant

```suggestion
  - name: Match

```